src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldNotContainUnderscores.Fixer.cs

```diff
@@ -19,11 +25,33 @@ public sealed override FixAllProvider GetFixAllProvider()
             return WellKnownFixAllProviders.BatchFixer;
         }
 
-        public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
+        public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
-            // Fixer not yet implemented.
-            return Task.CompletedTask;
+            var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+            var model = await context.Document.GetSemanticModelAsync(context.CancellationToken).ConfigureAwait(false);
 
+            var node = root.FindNode(context.Span);
+            var symbol = model.GetDeclaredSymbol(node, context.CancellationToken);
+
+            string title = MicrosoftCodeQualityAnalyzersResources.IdentifiersShouldNotContainUnderscoresTitle;
```

Comment:
We should use a different and more succint title for code fix, like say `Remove underscores`

Reply:
@mavasani, I followed the structure from another CodeFix, it used the same title.

![image](https://user-images.githubusercontent.com/31348972/83560500-a687c280-a516-11ea-8c2a-6d639e239ed5.png)

Is there something different here to use a different title unlike the other code fixes?

Should I add the fix title to the resources or just as private const ?

Reply:
> Is there something different here to use a different title unlike the other code fixes?

Fix title should ideally indicate the action being performed, rather then highlight the underlying issue. We also try to keep it smaller as it shows up in the light bulb.

> Should I add the fix title to the resources or just as private const ?

Yes, please add it the resources. All user facing strings should be added to resources file. Thanks!